source/includes/_payment_routing.md

```diff
@@ -674,10 +678,11 @@ Once user successfully do the payment, our system will make a callback via HTTP
 |need_frontend|Boolean|Partner need UI or not, if true, we will route to payment link, otherwise will be route to va aggregator.|
 |payment_info|Object|Payment info Object|
 |payment_checkout_url|String|generated url for payment link; conditional, only exist if request need_frontend is TRUE|
-|va_number|String|Generated VA number; conditional, only exist if request need_frontend is FALSE and payment_method is VA|
-|va_display_name|String|VA display name; conditional, only exist if request need_frontend is FALSE and payment_method is VA|
+|account_number|String|Generated VA number if you use VA or account number destination if you use unique code BCA; conditional, only exist if request need_frontend is FALSE and payment_method is VA|
+|account_name|String|VA display name if you use VA or bank account's owner name if you use unique code BCA; conditional, only exist if request need_frontend is FALSE and payment_method is VA|
```

Comment:
same comment as above, mikir dulu

Reply:
same as above, kita leave as is dulu. nanti kalo ternyata client bingung baru kita cari wording lain deh